src/plugin/cesium/tiles/cesium3dtileslayer.js

```diff
@@ -65,10 +70,25 @@ plugin.cesium.tiles.Layer.prototype.synchronize = function() {
 
   if (!this.hasError()) {
     var tilesetUrl = '';
-    if (!isNaN(this.assetId)) {
+    if (!isNaN(this.assetId) && this.accessToken) {
       tilesetUrl = Cesium.IonResource.fromAssetId(this.assetId, {
         accessToken: this.accessToken
       });
+      tilesetUrl.then(() => {
+        console.log('Access token accepted');
+      }, () => {
+        console.log('Access token rejected');
+
+        this.error = "The provided access token was rejected. Turn the layer off and back on to provide another access token."
```

Comment:
This class extends `plugin.cesium.Layer`, which has a `getErrorMessage` function you can extend to provide this error message. In `plugin.cesium.Layer.prototype.synchronize`, that function is called and if it returns an error message the layer will dispatch the property change event to update the layer node. That means you shouldn't need to duplicate the dispatch logic here, or elsewhere in the class. Instead, you should be able to extend `getErrorMessage` to provide an appropriate message when `assetId` is defined and `accessToken` is not (these go hand-in-hand to access a remote Cesium Ion resource).